File: solutions/arvindsin02/Shape.java
package solutions.arvindsin02;

class Shape {
    double height;
    double width;

    Shape() {
        height = 0.0d;
        width = 0.0d;
    }
    Shape(double height, double width) {
        this.height = height;
        this. width = width;
    }

    double compute_area() {
        return height*width;
    }


}

File: solutions/Mangai94/Shape.cs

namespace sample
{
    public class Shape
    {
        public double height;
        public double width;

        public Shape(double height, double width)
        {   
            this.height = height;
            this.width = width;
        }

        public virtual double compute_area()
        {
            return 0;
        }
    }
}

